### DISCUSSION TITLE
请问支持知识库的保存和移植吗？

### DISCUSSION THREAD

**Discussion by Navigate4** [netease-youdao/QAnything#460] [Q&A]
我在服务器上运行了QAnything，并建立了一个知识库，如果我想在另一个服务器上部署，可以把原服务器运行的QAnything的知识库移植过去吗？如果可以，我该怎么做？

**Comment by TatyOko28** [comment]
Yes, it is possible to migrate a knowledge base created in QAnything from one server to another. You need to:

Locate the knowledge base files or database on the original server.
Copy the knowledge base to the target server.
Reconfigure QAnything on the new server to use the migrated knowledge base.

**Comment by yuzhuoheng** [comment]
这个问题 有解决吗
